assets/js/state/sagas/softwareUpdatesSettings.js

```diff
@@ -30,6 +31,10 @@ export function* fetchSoftwareUpdatesSettings() {
     const response = yield call(getSettings);
     yield put(setSoftwareUpdatesSettings(response.data));
   } catch (error) {
+    const errorCode = get(error, ['response', 'status']);
+    if (errorCode === 500) {
```

Comment:
I'm not very familiar with the responses from SUSE Manager. Is this the only `5XX` response it could return?

Reply:
I just handled a 500 from our server when it comes to fetching SUSE Manager's settings, not venturing further :+1: